cmd/query/app/http_handler.go

```diff
@@ -433,10 +439,21 @@ func (aH *APIHandler) getTrace(w http.ResponseWriter, r *http.Request) {
 	if !ok {
 		return
 	}
-	// TODO: add start time & end time
 	query := spanstore.GetTraceParameters{
 		TraceID: traceID,
 	}
+	startTime := r.FormValue(startTimeParam)
+	if startTime != "" {
+		// No need to handle err here, since time window is optional
```

Comment:
error handling is not about param being optional but about syntax errors. If we cannot parse the timestamp we need to let the user know why.

Reply:
done